app/views/questions/new.html.erb

```diff
@@ -5,23 +5,74 @@
+                      <h4 class="modal-title" id="myModalLabel">Choose Qset</h4>
+                    </div>
+                    <div class="modal-body">
+                      <!-- displays qsets in hierarchical order -->
+                      <div class="list-group">
+                        <% @qsets.walk_tree do |qset, level| %>
+                          <% if level > 0 %>
+                            <% level_viz = '&nbsp;&nbsp;&nbsp;&nbsp;' * (level - 1) %>
+                            <% if qset.settings(:permissions).qset_type == 'subsets' %>
+                              <a href="#" class="list-group-item disabled">
+                                <%= level_viz.html_safe %>
+                                <span class="glyphicon glyphicon-folder-close"></span>
+                                <p class="qset-name-modal inline-p">
+                                  <%= '&nbsp;&nbsp;'.html_safe %>
+                                  <%= qset.name %>
+                                </p>
+                              </a>
+                            <% else %>
+                              <a href="#" class="list-group-item qset-selectable">
+                                <%= level_viz.html_safe %>
+                                <span class="glyphicon glyphicon-list-alt"></span>
+                                <p class="qset-name-modal inline-p">
+                                  <%= '&nbsp;&nbsp;'.html_safe %>
+                                  <%= qset.name %>
+                                </p>
+                                <p class="qset-id-modal">
+                                  <%= qset.id %>
+                                </p>
+                              </a>
+                            <% end %>
+                          <% end %>
+                        <% end %>
+                      </div>
+                    </div>
+                    <div class="modal-footer">
+                      <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                      <button type="button" class="btn btn-primary">Select</button>
+                    </div>
+                  </div>
+                </div>
               </div>
               <%= f.fields_for :answers do |a| %>
-                <div class="form-group">
-                  <%= a.text_area :text, placeholder: 'Answer',  class: 'form-control' %>
-                </div>
+                <%= a.text_area :text, placeholder: 'Answer',  class: 'form-control' %>
               <% end %>
-              <div class="form-group">
-                <div class="center btn-div">
-                  <%= f.submit "Submit question", class: 'btn btn-theme btn-lg submit-question' %>
-                </div>
+              <div class="center btn-div">
+                <%= f.submit "Submit question", class: 'btn btn-theme btn-lg submit-question' %>
               </div>
             <%end%>
         </div>
```

Comment:
I double-checked the empty line on this one too. And I agree on your second point; the display can definitely be improved. I will work on it and add a new commit!